internal/cli/config/config.go

```diff
@@ -42,6 +44,10 @@ func Load() (*Config, error) {
 	v.SetDefault("epinio_protocol", "http")
 	v.SetDefault("org", "workspace")
 
+	// The defaults are a non-working user/pass combination
```

Comment:
what does "non-working" mean here? Maybe change it to `Use empty defaults in viper to allow NeededOptions defaults to apply` ?

Reply:
I changed the text and updated the commit.